config/.env.base.sh

```diff
@@ -1,5 +1,5 @@
 ## Common
-export API_HOST='http://localhost:8009'
+export API_HOST='http://api-public:3000'
```

Comment:
not sure we should use the api-public dns entry as it's not documented here
```suggestion
# Public API entry point
export API_HOST='http://api-public:3000'
```

Reply:
see comment above, now the doc point to our backend documentation that stated to configure the DNS entry